numpy/polynomial/__init__.py

```diff
@@ -20,6 +20,54 @@
 from .hermite_e import HermiteE
 from .laguerre import Laguerre
 
+def set_default_printstyle(style):
+    """
+    Set the default format for the string representation of polynomials.
+
+    Values for ``style`` must be valid inputs to ``__format__``, i.e. 'ascii'
+    or 'unicode'.
+
+    Parameters
+    ----------
+    style : str
+        Format string for default printing style. Must be either 'ascii' or
+        'unicode'.
+
+    Notes
+    -----
+    The default format depends on the platform: 'unicode' is used on 
```

Comment:
Remove trailing whitespace.
```suggestion
    The default format depends on the platform: 'unicode' is used on
```